osu.Game/Screens/OnlinePlay/Lounge/Components/DrawableRoom.cs

```diff
@@ -112,114 +133,209 @@ public DrawableRoom(Room room)
+                                                Origin = Anchor.CentreLeft
+                                            },
+                                        }
                                     },
                                     new FillFlowContainer
                                     {
-                                        Anchor = Anchor.BottomLeft,
-                                        Origin = Anchor.BottomLeft,
-                                        RelativeSizeAxes = Axes.X,
-                                        AutoSizeAxes = Axes.Y,
+                                        Anchor = Anchor.CentreLeft,
+                                        Origin = Anchor.CentreLeft,
+                                        AutoSizeAxes = Axes.Both,
                                         Direction = FillDirection.Vertical,
-                                        Spacing = new Vector2(0, 5),
                                         Children = new Drawable[]
                                         {
-                                            new RoomStatusInfo(),
-                                            new BeatmapTitle { TextSize = 14 },
-                                        },
+                                            new RoomNameText(),
+                                            new RoomHostText()
+                                        }
                                     },
-                                    new ModeTypeInfo
+                                    new FillFlowContainer
                                     {
-                                        Anchor = Anchor.BottomRight,
-                                        Origin = Anchor.BottomRight,
-                                    },
+                                        Anchor = Anchor.BottomLeft,
+                                        Origin = Anchor.BottomLeft,
+                                        AutoSizeAxes = Axes.Both,
+                                        Direction = FillDirection.Horizontal,
+                                        Spacing = new Vector2(4),
+                                        Children = new Drawable[]
+                                        {
+                                            new PlaylistCountPill
+                                            {
+                                                Anchor = Anchor.CentreLeft,
+                                                Origin = Anchor.CentreLeft,
+                                            },
+                                            new StarRatingRangeDisplay
+                                            {
+                                                Anchor = Anchor.CentreLeft,
+                                                Origin = Anchor.CentreLeft,
+                                                Scale = new Vector2(0.85f)
+                                            }
```

Comment:
Has it been considered how this should display when there's only one map? right now it displays as both min and max, which feels slightly weird:
![CleanShot 2021-08-05 at 02 39 00](https://user-images.githubusercontent.com/22781491/128269074-72e0a386-9cce-4379-9090-9af87498003d.png)

I think displaying as a singular star rating display would look better (should do the same for `min.StarRating == max.StarRating` in general).